src/rebar_prv_dialyzer.erl

```diff
@@ -353,8 +353,16 @@ update_base_plt(State, BasePlt, Output, BaseFiles) ->
             build_plt(State, BasePlt, Output, BaseFiles)
     end.
 
+build_plt(State, Plt, _, []) ->
+    ?INFO("Building with no files in ~p...", [Plt]),
+    Opts = [{get_warnings, false},
+            {output_plt, Plt},
+            {apps, [erts]}],
+    _ = dialyzer:run([{analysis_type, plt_build} | Opts]),
+    _ = dialyzer:run([{analysis_type, plt_remove}, {init_plt, Plt} | Opts]),
```

Comment:
So that's a case where no files are found, and rather than doing a run that would fail to generate a PLT and crash further uses, this generates a valid empty one? If so, I'd love to have that in a comment so we know the rationale behind that function clause.